.github/workflows/ci.yml

```diff
@@ -1,5 +1,15 @@
 name: CI
 
+env:
+  # 7 GiB by default on GitHub, setting to 6 GiB
+  # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
+  NODE_OPTIONS: --max-old-space-size=6144
```

Comment:
Maybe we don't need to set this until we need it. I imagine the node tasks we do now doesn't use much memory too.
```suggestion
```